nexus/db-model/src/zpool.rs

```diff
@@ -29,22 +30,36 @@ pub struct Zpool {
 
     // The physical disk to which this Zpool is attached.
     pub physical_disk_id: DbTypedUuid<PhysicalDiskKind>,
+
+    /// How much storage to prevent Crucible from allocating for regions on a
```

Comment:
I'm thinking we could make this comment a little more clear as to what it represents.  How about these ideas:

```
The minimum amount of storage that crucible will set aside (not allocate from) on a zpool.
Other things may have already or may in the future allocate from that space.
```

```
For the total size of a given pool, this amount is reserved and crucible will never allocate regions from this.
```

```
Crucible will only allocate up to (TOTAL_POOL_SIZE) - `control_plane_storage_buffer` from a given zpool.  This space remains available for other resources and will not be allocated by crucible regions.
```

Reply:
changed it in 0bbf9a769, let me know what you think